mypy.ini

```diff
@@ -31,6 +37,3 @@ check_untyped_defs = True
 
 [mypy-certbot_dns_digitalocean.*]
 check_untyped_defs = True
-
```

Comment:
Looks like something happened with the merge here. These lines for nginx shouldn't be deleted.